pkg/operator/etcd_assets/bindata.go

```diff
@@ -791,23 +790,12 @@ ${COMPUTED_ENV_VARS}
         memory: 600Mi
         cpu: 300m
     readinessProbe:
-      exec:
-        command:
-        - /bin/bash
-        - -c
-        - |
-          set -xe
-
-          # Unix sockets are used for health checks to ensure that the pod is reporting readiness of the etcd process
-          # in this container. While this might seem unnecessary the use of SO_REUSEADDR has made this explicitly
-          # required as the kernel will allow the reuse of a port while in TIME_WAIT. etcd requires socket
-          # path in this format <name>:<port> so port 0 is used only to meet this requirement.
-          unset ETCDCTL_ENDPOINTS
-          /usr/bin/etcdctl \
-            --command-timeout=2s \
-            --dial-timeout=2s \
-            --endpoints=unixs://${NODE_NODE_ENVVAR_NAME_IP}:0 \
-            endpoint health -w json | grep \"health\":true
+      httpGet: 
+        port: 9980
+        path: readyz
```

Comment:
Just figured out why the new guard pods were failing their readiness check.
Apparently the static pod controller guard pods are hardcoded to use `/healthz` instead of `/readyz` as the readiness endpoint check.
https://github.com/openshift/library-go/blob/master/pkg/operator/staticpod/controller/guard/manifests/guard-pod.yaml#L44

I thought the former was deprecated but I think I'll have to change this to `healthz`.
Or probably I can just keep this, and just also serve the `/healthz` endpoint for the guard pod.

Reply:
They don't. I just meant the new ones were failing when they were querying the older `/healthz` endpoint. Although that's fixed now since I've added the handler for both.